### PR TITLE
Add FP endianness point to 'Divergences from upstream'

### PR DESCRIPTION
**Pull request by vhscampos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -202,6 +202,7 @@ $ build.py --host-toolchain mingw --copy-runtime-dlls no
 ### newlib:
 * clang does not support naked attribute on a C function, breaking the linux startup (out of scope).
 * Target triple ending with eabi is not considered an ELF target.
+* Only in 0.1: set the correct floating-point endianness when building with Clang.
```
Comment (amilendra) [inline README.md:205]: Maybe reword it as "revision 0.1 only"?

Comment (vhscampos) [reply README.md]: Done

**Review by amilendra — APPROVED** [review]
